velox/exec/HashJoinBridge.h

```diff
@@ -32,13 +32,35 @@ class HashJoinBridgeTestHelper;
 /// the same name.
 class HashJoinBridge : public JoinBridge {
  public:
+  struct SpillResult {
```

Comment:
HashJoinTableSpillResult and move this out of HashJoinBridge?

Reply:
Due to the reason of not being able to move spillTable,  and that SpillResult is only used in this class, we can make it in anonymous namespace in .cpp.